pom.xml

```diff
@@ -206,4 +206,24 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <profiles>
+    <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+    </profile>
+    <profile>
+      <id>dev</id>
+      <modules>
+        <module>benchmark</module>
```

Comment:
I think it would be clearer to put the default modules under the default profile, to avoid overriding them.